overlay/triplets/x64-windows.cmake

```diff
@@ -0,0 +1,11 @@
+set(VCPKG_TARGET_ARCHITECTURE x64)
+set(VCPKG_CRT_LINKAGE dynamic)
+
+# Oscpack does not allow dynamic linking on Windows 
+# because of missing __declspec(dllexport) decorations 
+if(${PORT} MATCHES "oscpack")
+    set(VCPKG_LIBRARY_LINKAGE static)
+else()
+    set(VCPKG_LIBRARY_LINKAGE dynamic)
+endif()
```

Comment:
This is not the right approach. Instead please provide a patch with the missing __declspec(dllexport) declarations upstream and use dynamic linkage according our standards.

Reply:
I have not a big interest to do it now and I also think that a patch is a too fragile solution. 
It would be better to do this change via upstream.
Until this has happened, a static build is a good intermediate solution.

Reply:
It is the common procedure to fix build issues upstream microsoft/vcpkg this way. This shouldn't require an overlay in our repo. An overlay is only appropriate if the upstream rejects a patch.